projects/extension/src/background/index.ts

```diff
@@ -21,9 +27,15 @@ chrome.runtime.onInstalled.addListener(() => {
   const westendChainSpec = JSON.stringify(westend);
   const kusamaChainSpec = JSON.stringify(kusama);
   const polkadotChainSpec = JSON.stringify(polkadot);
-  manager.addSmoldot('westend', westendChainSpec).catch((e) => { console.error('Error creating westend smoldot: ', e); });
-  manager.addSmoldot('kusama', kusamaChainSpec).catch((e) => { console.error('Error Creating kusama smoldot', e); });
-  manager.addSmoldot('polkadot', polkadotChainSpec).catch((e) => { console.error('Error Creating polkadot smoldot', e); });
+  manager.addSmoldot('westend', westendChainSpec).catch((e) => { 
```

Comment:
Its probably better to create a map of name/specs and loop through it

Reply:
I think this is nice and clear even if it's a bit repetitive and we don't intend to add more networks to this list

Reply:
I still think that its better to loop through it

Reply:
It's s stylistic argument and it's got nothing to do with this pull request: the only change here was reformatting. I think this pull request is big enough already